File: HELLO_PYTHON/day06/mysql_delete.py
import pymysql
conn = pymysql.connect(
    host='127.0.0.1',
    user='root', 
    password='python', 
    db='python', 
    charset='utf8',
    port = 3305
)
cur = conn.cursor()

sql = f"""
    DELETE FROM emp 
    WHERE e_id = '6'
    """
cnt = cur.execute(sql)

print(cnt)

conn.commit()

cur.close()
conn.close()

File: HELLO_NODE/mysql_delete.js
const mysql = require('mysql');  // mysql 모듈 로드
const conn = {  // mysql 접속 설정
    host: 'localhost',
    port: '3305',
    user: 'root',
    password: 'python',
    database: 'python'
};

var e_id = '3';

let connection = mysql.createConnection(conn); // DB 커넥션 생성
connection.connect();   // DB 접속

sql = `delete from emp where e_id='${e_id}'`;
 
connection.query(sql, function (err, results, fields) { 
    if (err) {
        console.log(err);
    }
    console.log(results.affectedRows);
});
 
 
connection.end(); // DB 접속 종료